test/Groups/group_characters.jl

```diff
@@ -0,0 +1,132 @@
+@testset "show and print character tables" begin
+  io = IOBuffer();
+  t_a4 = character_table(alternating_group(4))
+  t_a5 = character_table("A5")
+
+  # `print` shows an abbrev. form
+  print(io, t_a4)
+  @test String(take!(io)) == "character_table(Alt( [ 1 .. 4 ] ))"
+
+  # default `show`
+  show(io, t_a4)
+  @test String(take!(io)) == "Alt( [ 1 .. 4 ] )\n\n 2  2  2       .       .\n 3  1  .       1       1\n                        \n   1a 2a      3a      3b\n2P 1a 1a      3b      3a\n3P 1a 2a      1a      1a\n                        \nχ₁  1  1       1       1\nχ₂  1  1 -ζ₃ - 1      ζ₃\nχ₃  1  1      ζ₃ -ζ₃ - 1\nχ₄  3 -1       .       .\n"
```

Comment:
Alternatively you could test the `show` method via a doctest, that might be more helpful?